ci/test_python.sh

```diff
@@ -50,6 +41,19 @@ set +e
 # bulk sampler IO tests (hangs in CI)
 
 if [[ "${RUNNER_ARCH}" != "ARM64" ]]; then
+  rapids-logger "Generate Python testing dependencies"
```

Comment:
```suggestion
  rapids-logger "(cugraph_dgl) Generate Python testing dependencies"
```

I think we should add something to these messages to help differentiate them.